jobs/JGLOBAL_ANALYSIS_STATS

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "anlstat" -c "base anlstat"
+
+##############################################
+# Set variables used in the script
+##############################################
+
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+# Generate COM variables from templates
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_OBS COM_CHEM_ANALYSIS
+
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_SNOW_ANALYSIS COM_CONF
```

Comment:
```suggestion
YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COMIN_OBS COMOUT_CONF \
                                                                             COMIN_ATMOS_ANALYSIS \
                                                                             COMIN_MARINE_ANALYSIS \
                                                                             COMIN_CHEM_ANALYSIS \
                                                                             COMIN_SNOW_ANALYSIS
```

Reply:
assuming `COM_CONF` is used for output... we also will need `COMOUT_STAT` or something like that defined as to where to save the output files to